pkg/crdutil/crdutil.go

```diff
@@ -84,38 +92,47 @@ func EnsureCRDsCmd() {
 		log.Fatalf("Failed to create API extensions client: %v", err)
 	}
 
-	if err := walkCrdsDir(ctx, client.ApiextensionsV1().CustomResourceDefinitions()); err != nil {
+	if err := walkCRDs(ctx, client.ApiextensionsV1().CustomResourceDefinitions()); err != nil {
 		log.Fatalf("Failed to apply CRDs: %v", err)
 	}
 }
 
-// walkCrdsDir walks the CRDs directory and applies each YAML file.
-func walkCrdsDir(ctx context.Context, crdClient v1.CustomResourceDefinitionInterface) error {
-	for _, crdDir := range crdsDir {
+// walkCRDs walks the CRDs directory and applies each YAML file.
+func walkCRDs(ctx context.Context, crdClient v1.CustomResourceDefinitionInterface) error {
+	for _, crdDir := range files {
```

Comment:
I understand that `files` is global, but does it makes sense to pass it in as an argument instead?

Reply:
Same for the `recursive` bool